precovery/orbit.py

```diff
@@ -249,7 +249,7 @@ def approximately_propagate(
 
         Accuracy will decrease as timedelta increases.
         """
-        if timedelta <= 3.0 and self.ra_velocity < 1 and self.dec_velocity < 1:
+        if np.abs(timedelta) <= -1.0 and self.ra_velocity < 1 and self.dec_velocity < 1:
```

Comment:
Good catch to use `np.abs` - but why the change of the RHS to `<= -1`? It seems like that expression will always be `false` now.

Reply:
This is just a temporary change to disable the approximate propagator. Disabling the approximate propagation helped recover missing observations for some of the objects we were looking at. The more sophisticated approach would be to analyze the trade-off between healpixel size, approximate propagation time scale, and the predicted RA and Dec velocites. For the time being, however, using 2-body by default and then N-body propagations for the more precise propagations is thankfully fast enough.

Reply:
Got it. Let's just remove approximate propagation entirely, then, and use 2-body. That will mean a lot of code deletion - which is a very good thing.

Reply:
The code is fast enough for now but once we start indexing bigger datasets we may need to go back and look for faster methods. The linear propagation should work fine for time regimes that are small (hours to days, for most objects). Let's keep it for now so we have a starting place for future optimization.

Reply:
Okay - but I still think this could be clearer. It could be (for example) `if False and np.abs(timedelta) <= 1.0 and ...`, or at least could have a comment. The current thing just looks very odd, and future optimization might come in, like, two years, and by different people, so we should be clearer here.